### PR TITLE
Fix official build by shortening prerelease label length

### PR DESCRIPTION
**Pull request by joperezr**
cc: @layomia @safern 

Official builds are failing due to NuGet requiring prerelease label length to be < 20 chars. This change will make it so that's the case.

### CODE REVIEW

**Review by layomia — APPROVED** [review]
Thanks!

**Comment by joperezr** [comment]
Given this is a small change only in prerelease label, I'll go ahead and merge to unblock official builds without waiting for CI to finish.

**Pull request merged**
